### DISCUSSION TITLE
Redis 7 compatibility & timeline

### DISCUSSION THREAD

**Discussion by greenmato** [phpredis/phpredis#2394] [Q&A] [answered]
We are trying to upgrade our Redis clusters to v7.0. How likely do you think it would be for us to run into issues using `phpredis` v5.3.7? 

So far everything seems to work fine when testing. Our code is the same as with Redis 6, so we wouldn't need any new features coming from 7. I also noticed that in https://github.com/phpredis/phpredis/issues/2068 you mention that your test suite also passes.

Should we wait for `phpredis` v6.0.0, or just go for it? And is there any rough timeline of when v6.0.0 will be released?
Does anyone from the community have any experience running v5.3.7 with Redis 7?

Any comment would be very helpful even if it's just "we don't know"..

**Comment by remicollet** [comment, marked as answer]
> Should we wait for phpredis v6.0.0

6.0.0 and 6.0.1 are released

> Does anyone from the community have any experience running v5.3.7 with Redis 7?

Should not break anything, only missing some new commands
